cli/utils.js

```diff
@@ -161,34 +143,17 @@ const getOptionAsArgs = (options = {}) => {
 /**
  *  Get options for the command
  */
-const getCommand = (cmd, args, opts) => {
-    let options = [];
-    if (opts && opts.argument && opts.argument.length > 0) {
+const getCommand = (cmd, args = []) => {
+    let cmdArgs;
+    if (args && args.length > 0) {
```

Comment:
args is always set, no need for this check

Reply:
```suggestion
    if (args.length > 0) {
```